intellst-front/src/views/Login.vue

```diff
@@ -50,15 +66,36 @@ import { mapActions } from "vuex";
 
 export default {
   data: () => ({
+    status: null,
     user: {
       username: "",
       password: "",
     },
+    show1: false,
```

Comment:
please use "talking" variable name instead of show1

Reply:
Done